Pages/shop.html

```diff
@@ -24,27 +24,46 @@
                     });
             });
         </script>
+        <style>
+            #iframeContainer {
+                display: flex;
+                justify-content: center;
+                align-items: center;
+                height: 100vh; /* Full viewport height */
+            }
+            #zeffyIframe {
+                width: 80vw; /* 50% of the viewport width */
+                height: 90vh; /* 50% of the viewport height */
+            }
+        </style>
```

Comment:
You can add these directly in the `iframeContainer` and `zeffyIframe` element `class=""` property using Tailwind CSS to avoid extra style tags like these examples from the documentation:

[class="flex"](https://tailwindcss.com/docs/display#flex) instead of `display: flex;`
[class="h-screen"](https://tailwindcss.com/docs/height#viewport-height) instead of `height: 100vh;`

The search feature in Tailwind docs is pretty good, you should try it out if you'd like to keep using Tailwind!